content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,125 @@
+
+### PersistentVolume Health Monitor
+Physical Volumes (PV) are commonly used in applications to get local, file-based storage. They can be used in many different ways and help users migrate applications without needing to re-write storage backends.
+
+Kubernetes 1.21 has a new alpha feature which allows PVs to be monitored for health of the volume and marked accordingly if the volume becomes unhealthy. Workloads will be able to react to the health state to protect data from being written or read from a volume that is unhealthy.
+
+### Reducing Kubernetes Build Maintenance
+Previously Kubernetes has maintained multiple build systems. This has often been a source of friction and complexity for new and current contributors.
+
+Over the last release cycle, a lot of work has been put in to simplify the build process, and standardize on the native Golang build tools. This should empower broader community maintenance, and lower the barrier to entry for new contributors.
+
+## Major Changes
+
+### PodSecurityPolicy Deprecation
+In Kubernetes 1.21, PodSecurityPolicy is deprecated. As with all Kubernetes feature deprecations, PodSecurityPolicy will continue to be available and fully-functional for several more releases. PodSecurityPolicy, previously in the beta stage, is planned for removal in Kubernetes 1.25.
+
+What's next? We're developing a new built-in mechanism to help limit Pod privileges, with a working title of “PSP Replacement Policy”. Our plan is for this new mechanism to cover all the key PodSecurityPolicy use cases, and with greatly improved ergonomics and maintainability. To learn more, read [PodSecurityPolicy: Past, Present, and Future](#).
+
+### TopologyKeys Deprecation
+The Service field `topologyKeys` is now deprecated; all the component features that used this field were previously alpha, and are now also deprecated.
+We've replaced `topologyKeys` with a way to implement topology-aware routing, called topology-aware hints.  Topology-aware hints are an alpha featuee in Kubernetes 1.21. You can read more details about the replacement feature in [Topology Aware Hints](/docs/concepts/services-networking/topology-aware-hints/); the related [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/2433-topology-aware-hints/README.md) explains the context for why we switched.
+
+## Other Updates
+
+### Graduated to Stable
+
+* [EndpointSlice](https://github.com/kubernetes/enhancements/issues/752)
+* [Add sysctl support](https://github.com/kubernetes/enhancements/issues/34)
+* [PodDisruptionBudgets](https://github.com/kubernetes/enhancements/issues/85)
+
+### Notable Feature Updates
+
+* [External client-go credential providers](https://github.com/kubernetes/enhancements/issues/541) - beta in 1.21
+* [Structured logging](https://github.com/kubernetes/enhancements/issues/1602) - graduating to beta
```

Comment:
As per https://github.com/kubernetes/enhancements/issues/1602#issuecomment-811865043, it seems like Structured logging will graduate to beta in 1.22.

Reply:
Good catch @xmudrii

Reply:
Thank you @xmudrii & @sftim ! Have modified this in the latest commit :)